app/js/app/directives/content/Accordion.js

```diff
@@ -50,6 +50,7 @@ define([], function() {
 				scope.titleSuffixes = {};
 
 				scope.toggleChild = function(i) {
+					console.debug("toggle child");
```

Comment:
!!!

Reply:
Good spot, I'll do a search and take them out.